README.md

```diff
@@ -0,0 +1,114 @@
+ # Consensus protocol
+  Consensus protocol called Segregated Byzantine Agreement (SBA from hereon). SBA is a permission-less Proof-of-Stake-based consensus mechanism with statistical finality guarantees. SBA belongs to the committee-based PoS subcategory due to its use of committees to finalize a block for a given round.
+ 
+   SBA is permission-less, meaning that any eligible (eligibility properties are defined below) Dusk Network protocol participant is authorized to join and participate in the consensus. An eligible participant is defined with the following properties:
+ 
+- A participant with a pre-configured amount of DUSK locked as a stake (a.k.a Provisioner)
+- A participant, whose stake has a maturity of at least 2 epochs (a.k.a Eligible Provisioner)
+
+Other terms used below:
+- `Round` - A single SBA execution
+- `Round iteration` - All three phases (Selection, 1th Reduction and 2th Reduction) executed in a row.
+- `Committee` - A subset of Eligible Provisioners, extracted on performing `Deterministic Sortition`
+
+# Repository structure
```

Comment:
Not sure if 'structure' is the best title. Maybe 'contents' ?

At the same time, it would be nice to have the actual repo structure explained (maybe at a later time)

Reply:
`Contents` sounds too general to me. I'd stick with `Repository structure` for now as it highlights the main components the repo implements respectively we should maintain.